src/styles/partials/components/_accordion-dark.scss

```diff
@@ -0,0 +1,28 @@
+.dg_accordion.dg_section.dark {
+  .dg_accordion-btn:focus {
+    color: $white;
+  }
+  .dg_allitems {
+    border-bottom: 1px $white solid;
```

Comment:
Recommend changing 1px to use the sass variable: $border-width-small.

Reply:
Fixed